src/world/common/UnkFunc47.inc.c

```diff
@@ -0,0 +1,12 @@
+#include "common.h"
+#include "map.h"
+
+ApiStatus N(UnkFunc47)(Evt* script, s32 isInitialCall) {
```

Comment:
This is a distance-calculating function

Reply:
Should I name it `UnkDistCalcFunc` then, or do you have something more specific in mind?